ui/src/text.test.tsx

```diff
@@ -31,4 +32,15 @@ describe('Text.tsx', () => {
     expect(queryByTestId(name)).toBeInTheDocument()
   })
 
+  it('Renders Text with the align prop', () => {
+    const { queryByTestId, rerender } = render(<XText {...textProps} />)
+    expect(queryByTestId(name)).toBeInTheDocument()
+    expect(queryByTestId(name)).toHaveAttribute('style','text-align: start;')     
+    textAlignPropValues.forEach(textAlignPropValue => {
+      rerender(<XText {...textProps} align={textAlignPropValue} />)
+      expect(queryByTestId(name)).toBeInTheDocument()
+      expect(queryByTestId(name)).toHaveAttribute('style','text-align: '+textAlignPropValue+';')      
```

Comment:
TLDR: This unit test can be removed.

No need to unit test visual things as they are very coupled to the implementation - will break on refactor. We have visual regression tests for this scenario that can be used to verify no other components changed unexpectedly.

If you are interested, you can checkout to main, run `make build-ui` to build the UI, then `cd tools/showcase` and `make visual-regression`. If you have never run it before, it will create base snapshots that will be used for comparison in the next run. Now checkout your feature branch, build UI again and run visual regression tests again. It parses all the examples in documentation and compares against your baseline (main). It's super strict so false positives are very likely, but still good enough to quickly see if something broke or not. One more thing - if new examples are inserted (like it is now in `text.md`), the order will not match and you will get a lot of mismatches. In that case, you just check if the screenshots look "normal" since there is no way around that.

Reply:
@mturoci Thanks for explaining this about the visual regression tests, I hadn't noticed it yet! I ran through this and think this is a good testing tool. I'll be sure to run this as a check before submitting any UI changes in the future.